src/main/java/com/denizenscript/clientizen/objects/properties/item/ItemDurability.java

```diff
@@ -0,0 +1,46 @@
+package com.denizenscript.clientizen.objects.properties.item;
+
+import com.denizenscript.clientizen.objects.ItemTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+
+public class ItemDurability extends ItemProperty<ElementTag> {
+
+    // <--[property]
+    // @object ItemTag
+    // @name durability
+    // @input ElementTag(Number)
+    // @description
+    // The amount of durability an item lost.
```

Comment:
Match the Denizen documentation on this one (since this tag name has always been misleading 😭)

Reply:
Why match it? it's a lot less clear on the actual behavior imo

Reply:
if the meta doc for the same value in Denizen is bad, improve it there?

Reply:
I guess. It's always seemed to me like the `durability` tag is the *opposite* of what it's supposed to be, and the Denizen meta compensates for that

Reply:
I'd say blame Bukkit but I think the weirdness originates from Notch in this case, goes way back in time